app/tasks/maintenance/update_closing_reason_if_no_replaced_by_id_task.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module Maintenance
+  class UpdateClosingReasonIfNoReplacedByIdTask < MaintenanceTasks::Task
+    def collection
+      Procedure
+        .with_discarded
+        .closes
+        .where(closing_reason: Procedure.closing_reasons.fetch(:internal_procedure))
+        .where(replaced_by_procedure_id: nil)
+    end
+
+    def process(procedure)
+      procedure.update!(closing_reason: Procedure.closing_reasons.fetch(:other))
```

Comment:
de la même manière après le changement sur prefix, tu pourrais faire : 
```suggestion
      procedure.closing_reason_other!
```
ce qui fait un save/update derrière